src/tests/ftest/pool/attribute.yaml

```diff
@@ -12,9 +12,12 @@ pool:
   scm_size: 1073741824
   name: daos_server
 attrtests:
-  name_handles: !mux
+  test_handles: !mux
```

Comment:
This change disables the test's ability to mux together different combinations on pool attribute names and values, for example the previous version of this yaml file:
```
attrtests:
  name_handles: !mux
    valid:
      - 1
      - "PASS"
    invalid:
      - NULL
      - "FAIL"
  value_handles: !mux
    valid:
      - "ABCDEF"
      - "PASS"
    invalid:
      - NUL
      - "FAIL"
```

Would yield 4 test variants that will call pool `set_attr` each with one of the following combinations:
- {1: "ABCDEF"} => PASS
- {1: NULL} => FAIL
- {NULL: "ABCEDF"} => FAIL
- {NULL: NULL} => FAIL

Reply:
The original yaml file has the following behaviour:
name_handles: [0,1,2,3,4]
value_handles:[0,1,2,3,4]

The combination of tests is: (0,0), (0,1), (0,2), (0,3), (0,4), (1,0), (1,1,) .... (4,4). The only tests passing were (0,0), (1,1), (2,2), (3,3) and (4,4), which leads me to think the behaviour of the mux was not what was intended?

Reply:
Fixed